src/app/App.js

```diff
@@ -103,6 +104,13 @@ export default class App {
     log.info(` Admin for proxy ${proxyAddress} set to ${newAdmin}`)
   }
 
+  async createNonUpgradeableInstance(contractClass, contractName, initMethodName, initArgs) {
```

Comment:
How about `createContract` or `copyContract` instead? `nonUpgradeableInstance` does not sound very good, although I concur that it is very descriptive.